### PR TITLE
rss-bridge: init at 2020-02-26

### PR DESCRIPTION
**Pull request by dawidsowa**
###### Motivation for this change
Add [rss-bridge](https://github.com/RSS-Bridge/rss-bridge).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/web-apps/rss-bridge/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, fetchFromGitHub, stdenv, ... }:
```
Comment (jonringer) [inline pkgs/servers/web-apps/rss-bridge/default.nix]:
```suggestion
{ config, lib, pkgs, fetchFromGitHub, stdenv, ... }:

```

**Comment by jonringer** [comment]
@Infinisil do you mind taking a loot at the module?

**Comment by jonringer** [comment]
also, how can we test this?

**Comment by Nadrieril** [comment]
FWIW, I tested this and it works

File: pkgs/servers/web-apps/rss-bridge/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  installPhase = ''
+    mkdir $out/
+    cp -R ./* $out
+  '';
+
+  meta = with lib; {
+    description = "The RSS feed for websites missing it";
+    homepage = "https://github.com/RSS-Bridge/rss-bridge";
+    license = licenses.unlicense;
+    maintainers = with maintainers; [ dawidsowa ];
```
Comment (Infinisil) [inline pkgs/servers/web-apps/rss-bridge/default.nix:30]: `platforms` is missing

**Review by infinisil — APPROVED** [review]
Other than the `platform` missing, this looks good to me and is ready to merge!

**Pull request merged**